src/CommunityToolkit.Maui.MediaElement/MediaElement.shared.cs

```diff
@@ -10,6 +10,39 @@ namespace CommunityToolkit.Maui.Views;
 /// </summary>
 public partial class MediaElement : View, IMediaElement, IDisposable
 {
+	
+	/// <summary>
+	/// Constructor with Options
+	///</summary>
+	public MediaElement(MediaElementOptions? mediaElementOptions = null) 
+	{
+		// Assign default options (which may be set globally by UseMauiCommunityToolkitMediaElement in builder) if argument here is null
+		if (mediaElementOptions == null) { 
```

Comment:
for `null` check we use pattern matching, so you should use `is null` or `is not null` instead.

```suggestion
		if (mediaElementOptions is null) { 
```